experimental/PropertyDDS/examples/partial_checkout/src/demo/squaresApp.ts

```diff
@@ -65,7 +66,7 @@ export class SquaresApp {
         this.fluidBinder.defineRepresentation("view", "autofluid:squaresBoard-1.0.0", (property) => {
             const board =  new SquaresBoard([], this.container);
             // Rendering move button to move board's squares randomly
-            renderMoveButton(this.fluidBinder.getWorkspace(), board.wrapper, property.getId() as string);
+            renderMoveButton(this.fluidBinder.getWorkspace() as unknown as SharedPropertyTree, board.wrapper, property.getId() as string);
```

Comment:
NIT: Can't this just be `as SharedPropertyTree`  or `this.fluidBinder.getWorkspace()! as SharedPropertyTree`?

Reply:
No, Fluid complained that the two types are incompatible and cannot be directly casted.